src/js/utils/JSONUtil.js

```diff
@@ -0,0 +1,299 @@
+/**
+ * Tokenizer regular expressions
+ */
+const TOKENIZER_TOKENS = {
+  'comma': /,/g,
+  'end-label': /:/g,
+  'begin-object': /\{/g,
+  'end-object': /\}/g,
+  'begin-array': /\[/g,
+  'end-array': /\]/g,
+  'string': /"(\\["\\/bfnrtu"]|[^"\\"])*"/g,
+  'maybe-string': /"([^"]|\\")*/g,
+  'null': /null/g,
+  'boolean': /(true|false)/g,
+  'number': /-?\d+(\.\d+)?([eE]-?\d+)?/g,
+  'maybe-decimal-number': /-?\d+\./g,
+  'maybe-negative-number': /-/g,
+  'maybe-exponential-number': /-?\d+(\.\d+)?([eE])?/g,
+  'maybe-exponential-number-negative': /-?\d+(\.\d+)?([eE]-)?/g,
+  'symbol': /\w+/g
+};
+
+// Extract only keys
+const TOKENIZER_TYPES = Object.keys( TOKENIZER_TOKENS );
```

Comment:
Please rename this to `TOKENIZER_NAMES` or adjust the  argument name (`tokenName `) in line 34, for consistency.